src/main/java/org/opentripplanner/transit/raptor/rangeraptor/path/DestinationArrivalPaths.java

```diff
@@ -72,12 +74,16 @@ public boolean isReachedCurrentRound() {
         return reachedCurrentRound;
     }
 
+    public void setRangeRaptorIteration(int iteration) {
+        this.iteration = iteration;
+    }
+
     public boolean isEmpty() {
         return paths.isEmpty();
     }
 
     public boolean qualify(int departureTime, int arrivalTime, int numberOfTransfers, int cost) {
-        return paths.qualify(Path.dummyPath(departureTime, arrivalTime, numberOfTransfers, cost));
+        return paths.qualify(Path.dummyPath(iteration, departureTime, arrivalTime, numberOfTransfers, cost));
```

Comment:
We discussed the fact that this optimization cuts about 50% of the run time. It is the most effective optimization, and a paper came out in January explaining why it is so effective. Javadoc should be updated to include that reference.

Reply:
I have added some documentation and a reference to the paper.